src/css/_expand.scss

```diff
@@ -0,0 +1,38 @@
+$box-shadow: 0px 0px 12px rgba(88, 98, 109, 0.07), 0px 2px 4px rgba(88, 98, 109, 0.3);
```

Comment:
I think that it should be in another file but I don`t how give a name for it)

Reply:
If these shadows repeat across the project, create mixins and place them in separate `mixins.scss` file.